goal_src/jak2/engine/target/gun/gun-h.gc

```diff
@@ -5,5 +5,322 @@
+   (laser-dir              vector                :offset 320)
+   (laser-hit-point        vector                :offset 352)
+   (track?                 uint16                :offset 368)
+   (track-tilt             degrees               :offset 372)
+   (track-turn             degrees               :offset 376)
+   (track-find-range       meters                :offset 380)
+   (track-turnv-range      meters                :offset 384)
+   (track-tilt-range       meters                :offset 388)
+   (track-turn-range       meters                :offset 392)
+   (track-tilt-max         degrees               :offset 396)
+   (track-turn-max         degrees               :offset 400)
+   (track-angle-mult       float                 :offset 404)
+   (track-beam-size        float                 :offset 408)
+   (track-auto-fire        symbol                :offset 412)
+   (track-require          uint32                :offset 416)
+   (track-target-hold-time time-frame            :offset 424)
+   (track-start-time       time-frame            :offset 432)
+   (track-press-start-time time-frame            :offset 440)
+   (track-target           focus         2       :offset 448)
+   (track-trans            vector                :offset 480)
+   (track-dir              vector                :offset 496)
+   (turn-fast-hold-time    time-frame            :offset 512)
+   (blue-whine-sound-id    uint32                :offset 520)
+   (blue-whine-volume      float                 :offset 524)
+   (top-anim-twist         vector                :offset 528)
+   (top-anim-twist-targ    vector                :offset 544)
+   (top-anim-look-at       vector                :offset 560)
+   (top-anim-twist-reset   uint64                :offset 576)
+   (top-anim-gun-height    meters                :offset 584)
+   (top-anim-blue-cycle    float                 :offset 588)
+   (top-anim-low-high      float                 :offset 592)
+   (top-anim-extra-twistv  degrees               :offset 596)
+   (top-anim-tilt-up       degrees               :offset 600)
+   (attack-combo           combo-tracker         :offset 608)
+   (combo-window-start     time-frame            :offset 656)
+   (combo-window-state     uint32                :offset 664)
+   (combo-fire-delay       uint32                :offset 668)
+   (charge-ammo            float                 :offset 672)
+   (charge-start-time      time-frame            :offset 680)
+   (charge-inc-time        time-frame            :offset 688)
+   (charge-active?         uint64                :offset 696)
+   )
+  :method-count-assert 10
+  :size-assert         #x2c0
+  :flag-assert         #xa000002c0
+  (:methods
+    (gun-info-method-9 () none 9)
+    )
+  )
+
+(defun want-to-gun? ((arg0 target) (arg1 int))
+  "Are we able to pull out a gun?"
+  (local-vars (v1-36 symbol))
+  (with-pp
+    (the-as
+      symbol
+      (and (logtest? (-> arg0 game features) 32)
```

Comment:
I bet
```
(features               uint64               :offset-assert 128)
```
in game-info is a bitfield enum, but we dont' have to worry about that yet.

Reply:
Yep, I imagine it correlates to `Game... -> Features...` in the debug menu.